controls/slick.gridmenu.js

```diff
@@ -679,6 +679,7 @@
         let visibleColumns = [];
         columnCheckboxes.forEach((columnCheckbox, idx) => {
           if (columnCheckbox.checked) {
+            if (columns[idx].hidden) columns[idx].hidden = false;
```

Comment:
I'm not a fan of 1 liner without curly braces especially for assignment. I accept 1 liner for return but I would prefer this one wrapped in curly braces because it's assignment